js/web/lib/wasm/jsep/webgpu/ops/pool.ts

```diff
@@ -396,13 +401,13 @@ export const parseMaxPoolAttributes = (attributes: Record<string, unknown>): Max
   if (attr.ceilMode !== 0) {
     throw new Error('using ceil() in shape computation is not yet supported for MaxPool');
   }
-
-  return createAttributeWithCacheKey({storageOrder, dilations, ...attr});
+  return {storageOrder, dilations, ...attr, cacheKey: createShaderKeyFromAttributes(attr)};
 };
 
 export const parseGlobalMaxPoolAttributes = (attributes: Record<string, unknown>): MaxPoolAttributes => {
   const format = attributes.format as FormatAttributes['format'];
-  return {format, ...globalPoolAttributes, cacheKey: format};
+  const attributesWithoutCacheKey = {format, ...globalPoolAttributes};
+  return {...attributesWithoutCacheKey, cacheKey: createShaderKeyFromAttributes(attributesWithoutCacheKey)};
```

Comment:
for MaxPool, cache key should be format. because all other attributes are always empty.

Reply:
don